app/views/catalog/_previous_next_doc.html.erb

```diff
@@ -1,18 +1,19 @@
 <% #Using the Bootstrap Pagination class  -%>
-<% if @previous_document || @next_document %>
```

Comment:
Can you fix the indentation in this file? it seems  a little screwy (and probably not really your fault.)